eclass/perl-functions.eclass

```diff
@@ -157,7 +157,7 @@ perl_fix_packlist() {
                         einfo "Fixing packlist file /${f#${D}}"
 
 			# remove the temporary build dir path
-			sed -i -e "s:${D%/}/:/:g" "${f}"
+			sed -i -e "s:${D}/:/:g" "${f}"
```

Comment:
i suppose a `|| die` is missing here?

Reply:
Do have to be more careful about adding `|| die` in major eclasses, seds are sometime less deterministic and it could break some edge cases, and inspecting them may be bit out-of-context for this PR.

...albeit in this case `${f}` should always exist given it was just used by `file`, so I don't see how it could be an issue.

Reply:
(several die missing there though, and grep below needs special handling if we want to check return codes, so ignoring it here may be fair until someone properly looks at this)